smoke-tests/src/test/java/com/splunk/opentelemetry/GlassFishSmokeTest.java

```diff
@@ -33,8 +33,8 @@
 
   private static Stream<Arguments> supportedConfigurations() {
     return configurations("payara")
-        .otelLinux("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_ALL, "8", "11")
-        .otelWindows("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_ALL, "8", "11")
+        .otelLinux("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_HOTSPOT, "8")
+//        .otelWindows("5.2020.6", PAYARA_SERVER_ATTRIBUTES, VMS_ALL, "8", "11")
```

Comment:
?

Reply:
Remains of local testing, I forgot to remove that - thanks!